java/maven/arch.xml

```diff
@@ -186,6 +193,20 @@
           to specify the default behavior of compile on save in Maven 
           projects.
       </api> 
+      <api category="devel" group="property" name="exec.vmArgs" type="export">
```

Comment:
As this is going to be used in end-user projects configurations, I am afraid `devel` is a bit too low category in the long term.

Reply:
Changed to `stable` in 9d7654ce75